data/europe.yaml

```diff
@@ -165,12 +165,12 @@
   long: 8.42477
   address: Karlsruhe, Germany
   link: http://www.fzi.de/en/research/research-sectors/anwendungsfelder-en/afeld/automation-und-robotik/
-- name: Fatih BAYGÜL
-  type: other
```

Comment:
ROS Enthusiast is not a valid type just yet.

Reply:
I fixed it. Thanks

Reply:
@DLu can you merge it ?